.github/ISSUE_TEMPLATE/known-issue.yaml

```diff
@@ -0,0 +1,63 @@
+name: "Known issue"
+description: "Report a known issue for us to include in release notes."
+title: "[Known Issue] "
+labels: "known-issue, release-notes"
+body: 
+  - type: markdown
+    attributes:
+      value: |
+        Hello! Use this form to submit a known issue to the [@elastic/security-docs](https://github.com/orgs/elastic/teams/security-docs) team.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe the known issue and its impact on users.
```

Comment:
We should also ask if there's an issue or PR that needs to be linked to. Typically, there won't be, but it can't hurt to ask if that's not the case.